### PR TITLE
added dashboard will adjust it tom. to put the image in place and make it look nice.

### PR DESCRIPTION
**Pull request by DarthDARKFather**
Tomorrow I will add more stuff to the user dashboard and make it look better, but for now this is ready to be merged.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -28,6 +28,7 @@
           </ul>
           <ul class="nav navbar-nav navbar-right">
           <% if user_signed_in? %>
+            <li><%= link_to 'My Profile', user_path(current_user) %></li>
```
Comment (kenmazaika) [inline app/views/layouts/application.html.erb]: It looks like the user_id is passed through the URL, but the value of it is disregarded and `current_user` is used instead.  I think we should either not pass through the user_id in the URL or we should show the dashboard for the user that is passed along.  Probably the second.

File: spec/controllers/users_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
```
Comment (kenmazaika) [inline spec/controllers/users_controller_spec.rb]: We probably should add a test here.

**Comment by kenmazaika** [comment]
Made a couple of comments.  If you're planning on addressing them in the next PR I'm happy to merge this for now.

**Comment by DarthDARKFather** [comment]
It can wait, I'll just work on it some more tonight and tomorrow now that I see the other issues you pointed out, thanks Ken :)!

**Comment by kenmazaika** [comment]
Awesome!  Looking forward to seeing the progress.

**Pull request closed without merging**